src/transcoding/codec/codecs/libs/vaapi.c

```diff
@@ -454,8 +455,12 @@ tvh_codec_profile_vaapi_h264_open(tvh_codec_profile_vaapi_t *self,
         case 0:
             // Uncontrained --> will allow any combination of parameters (valid or invalid)
             // this mode is usefull fur future platform and for debugging.
-            if (self->desired_b_depth) {
-                AV_DICT_SET_INT(opts, "b_depth", self->desired_b_depth, AV_DICT_DONT_OVERWRITE);
+            if (self->low_power == 0) {
+                if (self->desired_b_depth) {
```

Comment:
These 2 conditions can be combined using `&&`

Reply:
done